### PR TITLE
First VS Single

### PR DESCRIPTION
**Pull request by maktub82**

### CODE REVIEW

File: gallardo/first-vs-single/first-vs-single.md [hunk 1]
```diff
@@ -0,0 +1,114 @@
+¡Hola a todos! Linq se ha convertido en nuestro mejor amigo cuando trabajamos con colecciones de datos. Pese a la comodidad que nos ofrece, Linq no es solo azúcar sintáctico: es mucho más, pero ¿siempre usamos Linq correctamente?
```
Comment (danimart1991) [inline gallardo/first-vs-single/first-vs-single.md]: I think that you must put "Linq library" at some point indicating what is Linq. Or a short description.

File: gallardo/first-vs-single/first-vs-single.md [hunk 2]
```diff
@@ -0,0 +1,114 @@
...
+
+Hoy vamos a comparar `Single` y `First` para ver las diferencias y si los usamos correctamente.
```
Comment (danimart1991) [inline gallardo/first-vs-single/first-vs-single.md]: "Hoy vamos a comparar los métodos `Single`..."

PD: I see you use your Mac here. That's "`" are suspicious.

**Comment by danimart1991** [comment]
Very good work here. 🙂

File: gallardo/first-vs-single/first-vs-single.md (hunk 2 above)
Comment (maktub82) [inline gallardo/first-vs-single/first-vs-single.md]: You must study markdown.

File: gallardo/first-vs-single/first-vs-single.md (hunk 1 above)
Comment (acnagrellos) [inline gallardo/first-vs-single/first-vs-single.md]: SG: I don't like to start the post with "¡Hola a todos!". The intro without this sentence is enought

File: gallardo/first-vs-single/first-vs-single.md (hunk 2 above)
Comment (acnagrellos) [inline gallardo/first-vs-single/first-vs-single.md]: SG: As before, I don't like "Hoy vamos...", I prefer "Vamos a comparar".

File: gallardo/first-vs-single/first-vs-single.md [hunk 3]
```diff
@@ -0,0 +1,114 @@
...
+
+Escribí este post para investigar y sopesar si era verdad que no estaba utilizando `Single` en ocasiones que tendría que haberlo utilizado y creo que la respuesta es sí. En algunos desarrollos tendría que haberlo añadido y, si es el caso, controlar la excepción que lanza para ver cómo actuar al respecto.
+
+## Referencias
+
+* [Código]( https://github.com/dotnet/corefx/tree/master/src/System.Linq/src/System/Linq) de Linq.
+
+* [Código]( https://github.com/dotnet/coreclr/blob/master/src/mscorlib/src/System/Collections/Generic/List.cs) de List.
+
+¡Nos vemos en el futuro!
```
Comment (acnagrellos) [inline gallardo/first-vs-single/first-vs-single.md]: I like this farewell!

File: gallardo/first-vs-single/first-vs-single.md (hunk 2 above)
Comment (danimart1991) [inline gallardo/first-vs-single/first-vs-single.md]: Ok then.

**Pull request merged**